docs/UserGuide.md

```diff
@@ -422,5 +422,5 @@ Action     | Format, Examples
 **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List**   | `list`
 **Filter** | `filter CCA1, CCA2, ...` e.g. `filter NUS Cycling`
-**Assign** | `Assign INDEX /r ROLE` e.g. `Assign 2 /r Member`
+**Assign** | `Assign INDEX r/ ROLE` e.g. `Assign 2 r/ Member` or `Assign 2 r/Member`
```

Comment:
(I wonder if review comments count on PRs already closed)
But `Assign` should be `assign`.
This is already fixed in master.